src/chapter_3/the_accounts_struct.md

```diff
@@ -124,4 +126,41 @@ pub struct SetData<'info> {
 }
 ```
 
-You can find information about all constraints in the reference. We will cover some of the most important ones in the milestone project at the end of the Essentials section.
\ No newline at end of file
+You can find information about all constraints in the reference. We will cover some of the most important ones in the milestone project at the end of the Essentials section.
+
+## Safety checks
+
+Many of the [common security pitfalls](https://blog.neodyme.io/posts/solana_common_pitfalls) in Solana programs stem from the provision of arbitrary accounts to program functions. Checking the owner of an account, signer(s) of an instruction, or the account data type are examples. Anchor encourages checks through the use of the appropriate account types.
+
```

Comment:
The above parapragh feels redundant now that it has moved to this section. Although parts of it may be a good fit for the initial types introduction paragraph.

but for now we can remove this paragraph imo